code/game/objects/items/devices/silent_alarm.dm

```diff
@@ -0,0 +1,46 @@
+/obj/machinery/button/silent_alarm
+	name = "Silent alarm"
+	desc = "A small button that silently notifies security when pressed"
+	silent = TRUE
+	device_type = /obj/item/assembly/silent_alarm
+
+/obj/item/assembly/silent_alarm
+	name = "Silent alarm electronics"
+	desc = "The internal electronics of a silent alarm button"
+	icon_state = "control"
+	COOLDOWN_DECLARE(announce_cooldown)
+	assembly_flags = ASSEMBLY_SILENCE_BUTTON | ASSEMBLY_NO_DUPLICATES
+
+/obj/item/assembly/silent_alarm/activate(mob/user)
+	if(is_within_radio_jammer_range(src))
+		COOLDOWN_START(src, announce_cooldown, 1.5 MINUTES)
+		return
+	if(!COOLDOWN_FINISHED(src, announce_cooldown))
+		return
+	aas_config_announce(/datum/aas_config_entry/silent_alarm_trigger, list("LOCATION" = get_area_name(src)), null, list(RADIO_CHANNEL_SECURITY), "Message")
```

Comment:
I'd like to source being provided. It just checks if your button is in connected Z levels for now. I should add relays support for AAS later.
It'd be great for cases when your button is off-station and should not interact with them.

Also extracting channel to separate variable may be cool for admins to tweak with them or create syndie office buttons and etc.
```suggestion
	aas_config_announce(/datum/aas_config_entry/silent_alarm_trigger, list("LOCATION" = get_area_name(src)), src, list(broadcast_channel), "Message")
```